CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.h

```diff
@@ -39,9 +40,13 @@ namespace Esri::ArcGISRuntime {
   class UtilityTraceParameters;
 }
 
+namespace Esri::ArcGISRuntime::Authentication {
+  class ArcGISAuthenticationChallenge;
+}
```

Comment:
```suggestion
namespace Esri::ArcGISRuntime::Authentication
{
  class ArcGISAuthenticationChallenge;
}
```

Reply:
This is another cases of forward declare inconsistency. In all cases I tried to make the changes match any existing bracketing and indent style, rather than trying to reformat a bunch of additional code for conformance.

In this case, it matches the style of the similar block above.